<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>superadmin</class>
 <widget class="QMainWindow" name="superadmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/new/prefix1/icons/admin.png</normaloff>:/new/prefix1/icons/admin.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upadate">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>402</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新界面数据</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>组织架构</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>员工数据一览表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="ALL">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>部门管理</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionss"/>
    <addaction name="actionset"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>员工管理</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>员工部门管理</string>
     </property>
     <addaction name="actions1"/>
     <addaction name="actions2"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="actionreanew"/>
    <addaction name="actionedit_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>超级管理员设置</string>
    </property>
    <addaction name="actionRESi"/>
    <addaction name="actionREANEWACC"/>
    <addaction name="actioncode"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionss">
   <property name="text">
    <string>注册新部门</string>
   </property>
   <property name="iconText">
    <string>注册新部门</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;新建部门&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionset">
   <property name="text">
    <string>部门信息</string>
   </property>
   <property name="toolTip">
    <string>部门信息</string>
   </property>
  </action>
  <action name="actionRESi">
   <property name="text">
    <string>注册超级管理员账户</string>
   </property>
  </action>
  <action name="actionREANEWACC">
   <property name="text">
    <string>普通员工账号注册</string>
   </property>
  </action>
  <action name="actionde">
   <property name="text">
    <string>部门考勤管理</string>
   </property>
  </action>
  <action name="actionperson">
   <property name="text">
    <string>员工考勤管理</string>
   </property>
  </action>
  <action name="actionreanew">
   <property name="text">
    <string>注册新员工</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>员工离职</string>
   </property>
  </action>
  <action name="actionedit">
   <property name="text">
    <string>员工信息</string>
   </property>
  </action>
  <action name="actions1">
   <property name="text">
    <string>员工部门分配</string>
   </property>
  </action>
  <action name="actions2">
   <property name="text">
    <string>员工部门调换</string>
   </property>
  </action>
  <action name="actionedit_2">
   <property name="text">
    <string>员工信息</string>
   </property>
  </action>
  <action name="actioncode">
   <property name="text">
    <string>更改本账号密码</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
